2-Draft-Accepted/RFCXXXX-MSI-Installation-Path.md

```diff
@@ -0,0 +1,68 @@
+---
+RFC: RFC
+Author: Joey Aiello
+Status: Draft
+Version: 0.1
+Area: Installation
+Comments Due: March 27, 2018
+Plan to implement: Yes
+---
+
+# MSI Installation Path Location
+
+Today, when using the MSI package on Windows,
+PowerShell Core is installed to `$env:ProgramFiles\PowerShell\<version>\`.
+This path was chosen early in the development of PowerShell Core in order to provide a deterministic,
+file-based way to query the installed version of PowerShell.
+In this RFC, we will explore the possibility of installing to different paths in order to support certain side-by-side scenarios.
+
+## Motivation
+
+Now that PowerShell Core 6.0 has GA'd, some users would like to run a preview version of PowerShell Core 6.1 side-by-side with PowerShell Core.
+Today, that's only possible if one version is installed via the MSI and another is installed via the "portable" ZIP package.
+Unfortunately, this makes the ZIP-installed copy of PowerShell difficult to install, and
```

Comment:
> difficult to install, and

Maybe this should be changed to `difficult to install.`?

Reply:
Yeah, this looks like a sentence that may not have been going anywhere. I'll reword or remove.